src/oss/paths/legacy_authorizations_authID.yml

```diff
@@ -5,12 +5,16 @@ get:
   summary: Retrieve an authorization
   parameters:
     - $ref: "../../common/parameters/TraceSpan.yml"
-    - in: path
+    - in: query
```

Comment:
I think this change also requires updating the path in `src/oss.yml` so it doesn't have the trailing `/{authID}`